lib/driver.js

```diff
@@ -312,6 +308,20 @@ class XCUITestDriver extends BaseDriver {
       await this.opts.device.clearCaches('com.apple.mobile.installd.staging');
 
       await this.startSim();
+
+      if (this.opts.customSSLCert) {
+        if (await hasSSLCert(this.opts.customSSLCert, this.opts.udid)) {
+          log.info(`SSL cert ${_.truncate(this.opts.customSSLCert, {length: 20})} already installed`);
```

Comment:
it might be useful to quote cert content inside the log message(s)

Reply:
That's what the `${_.truncate(this.opts.customSSLCert, {length: 20})}` is for

Reply:
truncate will just cut the string off. I mean

`SSL cert '1234...'`

is, probably, easier to read than

`SSL cert 1234...`

Reply:
Oh I understand now, nm